8.cytopick_analysis/scripts/prediction_images_find.py

```diff
@@ -55,42 +52,62 @@ def random_cell_select(
 
 # parameters
 CELL_TYPE = "PBMC"
-feature = "Nuclei_Texture_SumVariance_CorrGasdermin_3_01_256"
 
 
 # In[4]:
 
 
+# Get the current working directory
+cwd = pathlib.Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
+
+# Check if a Git root directory was found
+if root_dir is None:
+    raise FileNotFoundError("No Git root directory found.")
+root_dir
+
+
+# In[5]:
+
+
+image_out_dir_path = pathlib.Path(f"{root_dir}/8.cytopick_analysis/figures/PBMC/")
+
+
+# In[6]:
+
+
 # define directories
 # where the images are
 image_dir_path = pathlib.Path(
     "/media/lippincm/18T/interstellar_data/70117_20230210MM1_Gasdermin514_CP_BC430856__2023-03-22T15_42_38-Measurement1/2.IC/"
-)
-# if path does not exist, create it
-image_dir_path.mkdir(parents=True, exist_ok=True)
+).resolve(strict=True)
```

Comment:
Where does this path come from? Consider adding a quick comment about where the data is sourced from if that's not already documented elsewhere.

Reply:
added documentation describing that this is a hardcoded path to a local machine drive in the root